pkg/stream/vcenter.go

```diff
@@ -0,0 +1,76 @@
+package stream
+
+import (
+	"context"
+	"net/url"
+
+	"github.com/pkg/errors"
+	"github.com/vmware-samples/vcenter-event-broker-appliance/vc-event-router/pkg/connection"
+	"github.com/vmware-samples/vcenter-event-broker-appliance/vc-event-router/pkg/processor"
+	"github.com/vmware/govmomi"
+	"github.com/vmware/govmomi/event"
+	"github.com/vmware/govmomi/vim25/soap"
+	"github.com/vmware/govmomi/vim25/types"
+)
+
+const (
+	authMethodvSphere = "user_password"
+)
+
+// vCenter handles the connection to vCenter to retrieve an event stream
+type vCenter struct {
```

Comment:
I know this is the style but we can go with `VCenter` and name the variables `vCenter`

Reply:
Definitely, if the intention is to make the struct public.

Reply:
`vCenterStream` following the xStream|Processor struct naming but unexported